templates/client.json.j2

```diff
@@ -1,7 +1,7 @@
 {
   "client": {
-    "name": "{{ ansible_hostname }}",
+    "name": "{{ sensu_client_name }}",
     "address": "{{ ansible_default_ipv4['address'] }}",
-      "subscriptions": {{ group_names | to_nice_json }}
+    "subscriptions": {{ sensu_client_subscriptions | to_nice_json(indent=6) }}
```

Comment:
Ah I seed what you mean now. I definitely agree it looks much nicer with the indentation, but I'm not ready to force people to use 2.2.X.